BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/SpanControl.cs

```diff
@@ -0,0 +1,17 @@
+namespace BugsnagUnityPerformance
+{
+    public interface IAppStartSpanControl
+    {
+        void SetType(string type);
+        void ClearType();
+    }
+    public static class SpanType
+    {
+        public static readonly AppStartQuery AppStart = new AppStartQuery();
+
+        public class AppStartQuery : ISpanQuery<IAppStartSpanControl> { }
```

Comment:
Just a thought, this could also be expressed as:
```suggestion
        public static readonly ISpanQuery<IAppStartSpanControl> AppStart = new AppStartQuery();

        internal class AppStartQuery : ISpanQuery<IAppStartSpanControl> { }
```

I'm not sure I'd call it "better", but hiding the `AppStartQuery` class might be an attractive option for now since it's hard to express that it shouldn't be instantiated.